lib/Cake/Test/Case/Model/Datasource/DboSourceTest.php

```diff
@@ -78,6 +78,14 @@ public function nestedTransactionSupported() {
 		return $this->useNestedTransactions && $this->nestedSupport;
 	}
 
+	public function writeQueryCache($sql, $data, $params = array()) {
+		return $this->_writeQueryCache($sql, $data, $params);
+	}
```

Comment:
@markstory Can we use `ReflectionProperty::setAccessible()` instead? As `_queryCache` is a protected property, we cannot make sure whether it being flushed or not in the test.